projects/ad777x_iio/scripts/requirements.txt

```diff
@@ -1,5 +1,5 @@
 numpy==1.22.4
-pyadi-iio==0.0.15
+pyadi-iio==0.0.12
```

Comment:
Can this update be mentioned in the commit msg?

Reply:
Yes, I have updated the message now